client/extensions/woocommerce/app/settings/email/email-settings/components/notifications-origin.js

```diff
@@ -20,7 +20,7 @@ const NotificationsOrigin = ( {
 	item,
 	recipient,
 	onChange,
-	isPlaceholder,
+	loading,
```

Comment:
Please add loading (or loaded) to the propTypes for this component and remove isPlaceholder